lib/ukallocbbuddy/bbuddy.c

```diff
@@ -472,6 +573,39 @@ static int bbuddy_addmem(struct uk_alloc *a, void *base, size_t len)
 			    (uintptr_t)a, min, (uintptr_t)(min + (1UL << i)),
 			    (i - __PAGE_SHIFT));
 
+		/*
```

Comment:
I really don't like having this within the allocator logic in `bbuddy_addmem`. I think that it should be in a separate function especially since there's a lot of duplicate code with other balloon-related stuff in other functions.